viewers/wicket/ui/src/main/java/org/apache/isis/viewer/wicket/ui/pages/bootstrap-overrides-all-v1.css

```diff
@@ -950,6 +950,10 @@ li.select2-results__option {
     padding-right: 15px;
 }
 
+.multiLineStringPanel .scalarValueInlinePromptLink textarea {
```

Comment:
I believe bootstrap-overrides-all-v1.css is no longer used, we might as well delete this file.